js/plates-view/plate-mesh.ts

```diff
@@ -404,6 +404,16 @@ export default class PlateMesh {
     if (this.store.renderBoundaries && field.boundary) {
       return BOUNDARY_COLOR;
     }
+    if (this.store.colormap === "age" && field.normalizedAge === MAX_NORMALIZED_AGE) {
+      // Note that this is special case for preexisting crust. Field color attribute will be used instead of
+      // regular crust age colormap (it's only applied to "fresh" crust)
+      return PREEXISTING_CRUST_COLOR;
+    }
+    // If crust age colormap should not be used, this would be an option. It avoid some strange color transitions,
+    // but provides more blurry and les precise rendering.
+    // if (this.store.colormap === "age") {
+    // return crustAgeColor(field.normalizedAge);
+    // }
```

Comment:
If you uncomment these three lines, we'll stick to vertex color attribute style of rendering. It can be useful to try to see and understand the difference (and/or show to the project team if they're not happy about the current result).

Reply:
Or maybe combine the two in some interesting way?